### PR TITLE
Version Packages

### LINKED ISSUE #4743: 🐛 BUG: 3.19.0 changed URL normalization
### Which Cloudflare product(s) does this pertain to?

Wrangler core

### What version(s) of the tool(s) are you using?

- affected versions: 3.19.0 - 3.22.4
- last working version: 3.18.0

### What version of Node are you using?

20.9.0

### What operating system are you using?

WSL Ubuntu 22.04.3 LTS

### Describe the Bug

Version 3.19.0  changed the  URL normalization (changing double slashes to the single slash) of the `request.url` string which is a BREAKING CHANGE.

Visiting `http://localhost:8787//test`:

 - v3.19.0: `http://localhost:8787/test`
 - v3.18.0: `http://localhost:8787//test`

Created simple "Hello world" app using `npx create-cloudflare@2.9.0`:

`wrangler.toml`:
```toml
name = "super-dew-c60b"
main = "src/index.js"
compatibility_date = "2023-07-17"
```

`src/index.js`:
```js
export default {
	async fetch(request) {
		return new Response(request.url);
	},
};
```

### Please provide a link to a minimal reproduction

_No response_

### Please provide any relevant error logs

_No response_

### PR DESCRIPTION
**Pull request by workers-devprod**
This PR was opened by the [Changesets release](https://github.com/changesets/action) GitHub action. When you're ready to do a release, you can merge this and the packages will be published to npm automatically. If you're not ready to do a release yet, that's fine, whenever you add more changesets to main, this PR will be updated.

# Releases
### Minor Changes

-   [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) [`dae30015`](https://github.com/cloudflare/workers-sdk/commit/dae30015c646502819d79bf8b8ae032c4aa0669d) Thanks [@gabivlj](https://github.com/gabivlj)! - Added `wrangler cloudchamber` commands

    See [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) for more details.

### Patch Changes

-   [#4674](https://github.com/cloudflare/workers-sdk/pull/4674) [`54ea6a53`](https://github.com/cloudflare/workers-sdk/commit/54ea6a53bd1f222308135ed96bbb16a019302382) Thanks [@matthewdavidrodgers](https://github.com/matthewdavidrodgers)! - Fix usage of patch API in bulk secrets update

    Only specifying the name and type of a binding instructs the patch API to copy the existing binding over - but we were including the contents of the binding as well. Normally that's OK, but there are some subtle differences between what you specify to _create_ a binding vs what it looks like once it's _created_, specifically for Durable Objects. So instead, we just use the simpler inheritance.

-   [#4772](https://github.com/cloudflare/workers-sdk/pull/4772) [`4a9f03cf`](https://github.com/cloudflare/workers-sdk/commit/4a9f03cf56c3041b5ad77a7d66f6458777d1e655) Thanks [@mrbbot](https://github.com/mrbbot)! - fix: ensure dev server doesn't change request URLs

    Previously, Wrangler's dev server could change incoming request URLs unexpectedly (e.g. rewriting `http://localhost:8787//test` to `http://localhost:8787/test`). This change ensures URLs are passed through without modification.

    Fixes #4743.

### Patch Changes

-   [#4717](https://github.com/cloudflare/workers-sdk/pull/4717) [`05e3b544`](https://github.com/cloudflare/workers-sdk/commit/05e3b5446390c41c9e1846714e4f6020ca34c490) Thanks [@dependabot](https://github.com/apps/dependabot)! - C3: Bumped `create-docusaurus` from `3.0.1` to `3.1.0`

-   [#4728](https://github.com/cloudflare/workers-sdk/pull/4728) [`38d656b0`](https://github.com/cloudflare/workers-sdk/commit/38d656b0108453ef9fafee6108c145a0abbcf95e) Thanks [@dependabot](https://github.com/apps/dependabot)! - C3: Bumped `create-svelte` from `6.0.5` to `6.0.6`

-   [#4769](https://github.com/cloudflare/workers-sdk/pull/4769) [`fd1e9aa9`](https://github.com/cloudflare/workers-sdk/commit/fd1e9aa96cc44308f2e1e6213b4da57ff9b53f10) Thanks [@dependabot](https://github.com/apps/dependabot)! - C3: Bumped `create-qwik` from `1.3.2` to `1.3.5`

-   [#4770](https://github.com/cloudflare/workers-sdk/pull/4770) [`f2bfa5e2`](https://github.com/cloudflare/workers-sdk/commit/f2bfa5e26a1f0e61f81ee699f04fa7ab88c1e315) Thanks [@dependabot](https://github.com/apps/dependabot)! - C3: Bumped `create-remix` from `2.4.1` to `2.5.0`

-   [#4776](https://github.com/cloudflare/workers-sdk/pull/4776) [`eefd232e`](https://github.com/cloudflare/workers-sdk/commit/eefd232efbfb3c57b8f03d205ae1831ee1a3f536) Thanks [@dependabot](https://github.com/apps/dependabot)! - C3: Bumped `create-astro` from `4.6.0` to `4.7.1`

### Minor Changes

-   [#4779](https://github.com/cloudflare/workers-sdk/pull/4779) [`06bb99e1`](https://github.com/cloudflare/workers-sdk/commit/06bb99e1c91ffe5305343a17353912e734f5bd0c) Thanks [@1000hz](https://github.com/1000hz)! - Added a new `SelectRefreshablePrompt` component

    This component supports infinite scrolling, and the ability to refresh the items on 'R' keypress. See [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) for more details.

-   [#4779](https://github.com/cloudflare/workers-sdk/pull/4779) [`06bb99e1`](https://github.com/cloudflare/workers-sdk/commit/06bb99e1c91ffe5305343a17353912e734f5bd0c) Thanks [@1000hz](https://github.com/1000hz)! - Added `@clack/core`'s `MultiSelectPrompt` component

    See [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) for more details.

-   [#4779](https://github.com/cloudflare/workers-sdk/pull/4779) [`06bb99e1`](https://github.com/cloudflare/workers-sdk/commit/06bb99e1c91ffe5305343a17353912e734f5bd0c) Thanks [@1000hz](https://github.com/1000hz)! - Added `processArguments` helper function

    See [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) for more details.

### Patch Changes

-   [#4779](https://github.com/cloudflare/workers-sdk/pull/4779) [`06bb99e1`](https://github.com/cloudflare/workers-sdk/commit/06bb99e1c91ffe5305343a17353912e734f5bd0c) Thanks [@1000hz](https://github.com/1000hz)! - Downgraded `chalk` dependency from `^5.2.0` to `^2.4.2`

    This was done for compatibility reasons with the version used in the `wrangler` package. See [#4310](https://github.com/cloudflare/workers-sdk/pull/4310) for more details.

### CODE REVIEW

**Review by 1000hz — APPROVED** [review]

**Review by lrapoport-cf — APPROVED** [review]

**Comment by lrapoport-cf** [comment]
pending all checks passing, this PR will be landed without waiting for approval by @cloudflare/c3 per https://github.com/cloudflare/workers-sdk/pull/4774.

**Pull request merged**
